cleanlab/datalab/datalab.py

```diff
@@ -188,6 +189,13 @@ def find_issues(
             To best detect label issues, provide this input obtained from the most accurate model you can produce.
 
             If provided, this must be a 2D array with shape (num_examples, K) where K is the number of classes in the dataset.
+            Must not be provided if `predictions` is provided.
+
+        predictions :
+            Out-of-sample predicted output variables made by the model for every example in the dataset.
+            To best detect label issues, provide this input obtained from the most accurate model you can produce.
+
+            Must not be provided if `pred_probs` is provided.
```

Comment:
needs more explanation about why this exists vs. just pred_probs and when to use one vs. the other

Reply:
what is long-term plan here? 

deprecate `pred_probs` and have all tasks use `predictions`?

If so, I'd do followup PR where you allow classification tasks to work with `predictions` as well, internally you could just set:
```
if predictions is not None:
    pred_probs = predictions 
```

Reply:
If ^ this is the long term plan, then will eventually need the `predictions` docstring to be quite involved for each task type. And should soon move the docstring for `pred_probs` into dosctring for `predictions` as well (in a subsection when the task is classification), so that `pred_probs` would start to be de-emphasized.

Alternatively if planning to keep `pred_probs` around for a while, want to think of plan on how to support both it and `predictions` in a nicer way

Reply:
We get the best of both worlds with the approach I briefly mention in https://github.com/cleanlab/cleanlab/pull/902#discussion_r1412323597.

We'll keep only `pred_probs` for backwards compatibility at the user-level.
The `RegressionLabelIssueManager` still uses the `predictions`  argument name, but these implementation details are handled in an intermediate stage when calling `Datalab.find_issues(pred_probs=...)`.
This gives us more time to think if we want to keep two, mutually exclusive arguments in `Datalab.find_issues` when one will suffice.

Docstring maintenance is also much simpler this way.